internal/runbits/requirements/requirements.go

```diff
@@ -23,10 +23,12 @@ import (
 	"github.com/ActiveState/cli/internal/prompt"
 	"github.com/ActiveState/cli/internal/runbits"
 	"github.com/ActiveState/cli/internal/runbits/rtusage"
+	bgModel "github.com/ActiveState/cli/pkg/platform/api/graphql/model/buildplanner"
```

Comment:
The name `bgModel` comes from when we were calling it a `build graph`. I did not update the naming after that. I've modified the names a bit but this might be something we want to revisit and reorganize. I will file a follow-up story to discuss.